core/src/main/java/org/apache/gravitino/hook/CatalogHookDispatcher.java

```diff
@@ -131,18 +131,18 @@ public boolean dropCatalog(NameIdentifier ident, boolean force)
       return false;
     }
 
-    // If we call the authorization plugin after dropping catalog, we can't load the plugin of the
-    // catalog
     Catalog catalog = dispatcher.loadCatalog(ident);
-    boolean dropped = dispatcher.dropCatalog(ident, force);
 
-    if (dropped && catalog != null) {
+    if (catalog != null) {
       List<String> locations =
           AuthorizationUtils.getMetadataObjectLocation(ident, Entity.EntityType.CATALOG);
       AuthorizationUtils.removeCatalogPrivileges(catalog, locations);
     }
 
-    return dropped;
+    // We should call the authorization plugin before dropping the catalog, because the dropping
+    // catalog
+    // will close the authorization plugin.
```

Comment:
?

Reply:
Correct. Gradle format will cause this issue.